src/codeu/chat/server/Controller.java

```diff
@@ -130,6 +146,19 @@ public User newUser(Uuid id, String name, Time creationTime) {
     return user;
   }
 
+  public User addPassword(User user, String password) {
```

Comment:
can we make this method void instead? it's not very intuitive that addPassword return a User